.github/workflows/e2e-tests.yml

```diff
@@ -398,6 +398,71 @@ jobs:
           output: |
             {"summary":${{ env.MOCHAWESOME_REPORT_RESULTS }}}
 
+  e2e-code-coverage-cypress:
+    name: E2E Code Coverage Reports
+    runs-on: ubuntu-latest
+    needs: [set-commit-sha, testing-reports-prep, cypress-tests-prep, cypress-tests]
+    continue-on-error: true
+    if: ${{ always() && needs.cypress-tests-prep.outputs.tests != '[]' && (needs.cypress-tests.result == 'success' || needs.cypress-tests.result == 'failure') }}
+    
+    env:
+      APPLICATION_LIST: ${{ needs.testing-reports-prep.outputs.app_list }}  
+
+    steps: 
+      - name: Checkout
```

Comment:
Do you need a specific `sha` here from `set-commit-sha`?

Reply:
Yes, that info would be nice to have. Added. Thanks @jhonnyoddball !